cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -1566,13 +1566,59 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
   });
 
   describe('invalid document structure', () => {
-    describe('normalizable errors', () => {
-      it('does not throw', () => {
-        cy.setInitialValue(invalidDocumentNormalizable);
-        cy.reload();
-        //check that editors content is what we expect (not a thrown error)
-        richText.expectSnapshotValue();
-      });
+    it('accepts document with no content', () => {
+      const emptyDocument = {
+        nodeType: 'document',
+        data: {},
+        content: [],
+      };
+
+      cy.setInitialValue(emptyDocument);
+
+      cy.reload();
+
+      // The field value in this case will still be untouched (i.e. invalid)
```

Comment:
Careful with words here as it's not "invalid" from the CMA's point of view but I guess you mean invalid in Slate terms? Perhaps

```suggestion
      // The field value in this case will still be untouched (i.e. unnormalized)
```